### ISSUE TITLE
Remove text for Hearing Impaired

### ISSUE THREAD

**Issue by mickeymichael123** [SubtitleEdit/subtitleedit#8653]
Hi,
When using "Remove text for hearing Impaired" it shows before and after, but when appied, it removes everthing and there is no after left?
I don't recall changing any settings?
I was using Version 4.05 so upgraded to 4.07 but the error persists.
Thanks
Mikey
![Screen Dump21-07-24-12 19 08 PM](https://github.com/user-attachments/assets/893e7164-ede4-4504-a049-20d6b2a98396)
![Screen Dump21-07-24-12 19 26 PM](https://github.com/user-attachments/assets/e8b037bc-88b8-4fda-a38c-b48de357ae4b)

**Comment by niksedk** [comment]
Yes... then it's fixed...

**Comment by mickeymichael123** [comment]
Hi niksedk - thanks for your prompt reply - the program is great.
I'm not sure that I explained my problem correctly.
I have added to the screen dup to try and make it clearer.
Thanks
![Remove text for Hearing Impaired · Issue #8653 · SubtitleEdit](https://github.com/user-attachments/assets/f9652f6c-895a-4a8a-ac3a-d8478f7d95cc)

**Comment by niksedk** [comment]
Works fine here... try to attach the subtitle file too, so I can give it a test.

**Comment by mickeymichael123** [comment]
Hey niksedk thanks for the reply.
I think it is my problem but I don't know what I have done.
Everything used to work just fine.
The problem is with every file that I try.
I have uninstalled (including settings) and reinstalled but to no effect - my problem remains.
I have tried to a attach a .srt file for you to test but it is not supported in the add files list, so I have changed the name to a text file. You will need to rename to .srt file.
[The Spy Who Came in from the Cold.txt](https://github.com/user-attachments/files/16368939/The.Spy.Who.Came.in.from.the.Cold.txt)
.
Thanks
mickeymichael
